src/Main.js

```diff
@@ -74,12 +74,31 @@ WebhookHandler.on('push', HandlerWrap((ev) => {
     }
 }));
 
+// prUrl format, e.g., https://api.github.com/repos/github/hello-world/pulls/1
+// whether the PR belongs to Squid Project
```

Comment:
Here and elsewhere, please adjust to remove an incorrect assumption that Anubis is Squid-specific. Also, IIRC, an Anubis instance monitors a repository rather than a project or organization.

I suggest using "monitored repository" and "foreign repository" terminology.

Reply:
Ok. The PR url contains the  'owner/repository' part and we filter out by the owner name (i.e., a person or an organization), such as 'squid-cache'. The repository name may be the same, i.e., 'squid'.